### DISCUSSION TITLE
Uptime Kuma 2.0.0-beta.0

### DISCUSSION THREAD

**Discussion by drueppela** [hassio-addons/addon-uptime-kuma#209] [General]
Is it possible to install [Uptime Kuma 2.0.0-beta.0 ](https://github.com/louislam/uptime-kuma/releases/tag/2.0.0-beta.0) in [Home Add-on Assistant Add-on Uptime Kuma](https://github.com/hassio-addons/addon-uptime-kuma)?

**Comment by frenck** [comment]
Nope, I ship only stable versions.

../Frenck
